packages/clarity-js/src/layout/dom.ts

```diff
@@ -297,10 +297,11 @@ export function hashText(hash: string): string {
 }
 
 export function getNode(id: number): Node {
```

Comment:
>getNode [](http://example.com/codeflow?start=16&length=7)

not sure it's any more efficient but feels a little more compact to have this be:

return nodesMap.has(id) ? nodesMap.get(id) : null